include/pm3_cmd.h

```diff
@@ -543,6 +543,8 @@ typedef struct {
 #define CMD_HF_THINFILM_READ                                              0x0810
 #define CMD_HF_THINFILM_SIMULATE                                          0x0811
 
+#define CMD_HF_ISO14443B_SIM_CRYPTORF                                     0x0812
```

Comment:
In order to keep things a bit clear,  lets follow the nomenclature:
CMD_HF_CRYPTORF_SIM
---------------------xxxxx
where xxxx will be the implemented command for cryptoRF.